data/fields/crossing/whistle.json

```diff
@@ -0,0 +1,15 @@
+{
+    "key": "crossing:whistle",
+    "type": "combo",
+    "label": "Train Horn",
```

Comment:
IMO it would be important to indicate that this tag should only be used in situations of _non-standard train horn operations_. :thinking: